FFXIVClientStructs/FFXIV/Client/UI/Misc/UIInputData.cs

```diff
@@ -0,0 +1,107 @@
+using FFXIVClientStructs.FFXIV.Client.System.Framework;
+
+namespace FFXIVClientStructs.FFXIV.Client.UI.Misc;
+
+[StructLayout(LayoutKind.Explicit, Size = 0xA20)]
+public unsafe partial struct UIInputData
+{
+    public static UIInputData* Instance() => Framework.Instance()->UIModule->GetUIInputData();
+
+    /*
+     * UIFiltered means those are not set if 
+     * - the game window is focused and
+     * - the cursor is hovering any interactable UI elements or windows
+     * 
+     * For mouse buttons, only Left and Right buttons are filtered out, extra buttons are not
+     */
+    [FieldOffset(0x498)] public int UIFilteredCursorXPosition;
+    [FieldOffset(0x49C)] public int UIFilteredCursorYPosition;
+    [FieldOffset(0x4A0)] public int UIFilteredMouseWheel; // -1 for scroll down, 1 for scroll up
+    [FieldOffset(0x4A4)] public MouseButtonFlags UIFilteredMouseButtonHeldFlags;
+    [FieldOffset(0x4A8)] public MouseButtonFlags UIFilteredMouseButtonPressedFlags;
+    [FieldOffset(0x4B0)] public MouseButtonFlags UIFilteredMouseButtonReleasedFlags;
+    [FieldOffset(0x4B4)] public MouseButtonFlags UIFilteredMouseButtonHeldThrottledFlags;
+    [FieldOffset(0x4B8)] public int UIFilteredCursorXDelta; // Delta since last frame
+    [FieldOffset(0x4BC)] public int UIFilteredCursorYDelta; // Delta since last frame
+
+    // Same as 0x4F4 ?
+    // [FieldOffset(0x4C4)] public byte IsGameWindowFocused;
+
+    [FieldOffset(0x4C8)] public int CursorXPosition;
+    [FieldOffset(0x4CC)] public int CursorYPosition;
+    [FieldOffset(0x4D0)] public int MouseWheel; // -1 for scroll down, 1 for scroll up
+    [FieldOffset(0x4D4)] public MouseButtonFlags MouseButtonHeldFlags;
+    [FieldOffset(0x4D8)] public MouseButtonFlags MouseButtonPressedFlags;
+    [FieldOffset(0x4E0)] public MouseButtonFlags MouseButtonReleasedFlags;
+    [FieldOffset(0x4E4)] public MouseButtonFlags MouseButtonHeldThrottledFlags;
+    [FieldOffset(0x4E8)] public int CursorXDelta; // Delta since last frame
+    [FieldOffset(0x4EC)] public int CursorYDelta; // Delta since last frame
+
+    // At least this is what it seems to be
+    [FieldOffset(0x4F4)] public byte IsGameWindowFocused;
```

Comment:
This should be a bool

Reply:
Right, Kami told me, forgot to change it. Thanks.